tests/plugin-test/css-extract/cases/pathinfo-devtool-source-map/expected/main.css

```diff
@@ -1,19 +1,19 @@
 /*!**********************************************************************************************************************************************************************************************************************!*\
-  !*** css ../../../../../node_modules/.pnpm/css-loader@7.1.2_@rspack+core@packages+rspack_webpack@5.94.0_@swc+core@1.10.1_@swc+helpers@0._sroe7k2leveebxngymnh5tghna/node_modules/css-loader/dist/cjs.js!./style.css ***!
+  !*** css ../../../../../node_modules/.pnpm/css-loader@7.1.2_@rspack+core@packages+rspack_webpack@5.95.0_@swc+core@1.10.1_@swc+helpers@0._fzyfl4cqgys6tkprvbceqky724/node_modules/css-loader/dist/cjs.js!./style.css ***!
```

Comment:
(It might be clever to normalize away the exact node_modules paths from these snapshots.)

Reply:
Agreed, we should add `path-serializer` for the `plugin-test` cases